### PR TITLE
[DS to LF] Modal

### PR DESCRIPTION
**Pull request by RipripKipkip**
## Description

Refactoring UI Modal

-----

image
image
image
image

- [x] Border to add in the header
- [x]  Box-shadow on the footer
- [x]  Padding to review
- [x]  Add new modal sizes
- [x]  Focus state on the cross
- [ ]  Mobile version to manage
- [ ]  Bonus : Add an outline: none; to lu-modal-header to avoid displaying focus style on some conditions (like open modal, push tab, close modal, open modal again. See screen bellow)

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1935/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1935/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1935/storybook

File: packages/ng/styles/components/_popup.scss [hunk 1]
```diff
@@ -20,13 +22,21 @@
...
-		width: var(--components-popup-S);
+		width: var(--components-popup-size-small);
 	}
 
 	&.mod-L {
 		width: var(--components-popup-L);
 	}
+
+	&.mod-XL {
+		width: var(--components-popup-size-larger);
```
Comment (AlexiaAlarza) [inline packages/ng/styles/components/_popup.scss:38]: var(--components-popup-XL)

File: packages/ng/styles/components/_popup.scss [hunk 2]
```diff
@@ -20,13 +22,21 @@
 	overflow: auto;
 	animation: popup var(--commons-animations-durations-fast) cubic-bezier(0.25, 0.8, 0.25, 1);
 
+	&.mod-XS{
+		width: var(--components-popup-size-smaller);
+	}
+
 	&.mod-S {
-		width: var(--components-popup-S);
+		width: var(--components-popup-size-small);
```
Comment (AlexiaAlarza) [inline packages/ng/styles/components/_popup.scss:30]:
```suggestion
		width: var(--components-popup-S);
```

File: packages/ng/styles/components/_popup.scss [hunk 3]
```diff
@@ -20,13 +22,21 @@
 	overflow: auto;
 	animation: popup var(--commons-animations-durations-fast) cubic-bezier(0.25, 0.8, 0.25, 1);
 
+	&.mod-XS{
+		width: var(--components-popup-size-smaller);
```
Comment (AlexiaAlarza) [inline packages/ng/styles/components/_popup.scss:26]:
```suggestion
		width: var(--components-popup-XS);
```

File: packages/ng/styles/components/_popup.scss [hunk 4]
```diff
@@ -42,9 +52,13 @@
...
 
 .lu-modal-header {
+	position: sticky;
+	top: 0;
+	z-index: 5;
+	background-color: var(--colors-white-color);
 	padding: var(--components-popup-padding-vertical) var(--spacings-XXL) var(--components-popup-padding-vertical)
 		var(--components-popup-padding-horizontal);
-	position: relative;
+	border-bottom: var(--commons-divider-width) solid var(--commons-divider-color);
```
Comment (AlexiaAlarza) [inline packages/ng/styles/components/_popup.scss:61]:
```suggestion
	border-bottom: var(--commons-divider-border);
```

File: packages/ng/styles/components/_popup.scss [hunk 5]
```diff
@@ -100,12 +96,16 @@
...
+	bottom: 0;
+	z-index: 5;
 	display: flex;
-	margin-top: auto;
 	align-items: center;
 	justify-content: flex-end;
+	background-color: var(--colors-white-color);
+	box-shadow: var(--commons-elevations-elevation-4);
+	margin-top: auto;
+	padding: var(--spacings-small) var(--components-popup-padding-vertical);
```
Comment (AlexiaAlarza) [inline packages/ng/styles/components/_popup.scss:108]:
```suggestion
	padding: var(--spacings-S) var(--components-popup-padding-vertical);
```

File: packages/ng/styles/components/_popup.scss [hunk 6]
```diff
@@ -54,32 +68,14 @@
...
-	padding: var(--spacings-XS);
-	height: 3rem;
-	width: 3rem;
 	position: absolute;
-	right: 0.6rem;
-	top: 0.6rem;
-	border-radius: 50%;
+	right: 1.25rem;
+	top: 0.75rem;
 	transition: background var(--commons-animations-durations-fast) ease;
```
Comment (AlexiaAlarza) [inline packages/ng/styles/components/_popup.scss:74]: à supprimer

File: packages/ng/styles/components/_popup.scss [hunk 7]
```diff
@@ -20,13 +22,21 @@
 	overflow: auto;
 	animation: popup var(--commons-animations-durations-fast) cubic-bezier(0.25, 0.8, 0.25, 1);
 
+	&.mod-XS{
+		width: var(--components-popup-XS);
+	}
+
 	&.mod-S {
-		width: var(--components-popup-S);
+		width: var(--components-popup-XS);
```
Comment (AlexiaAlarza) [inline packages/ng/styles/components/_popup.scss:30]:
```suggestion
		width: var(--components-popup-S);
```

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1935/storybook

**Review by AlexiaAlarza — APPROVED** [review]

**Review by jeremie-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1935/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1935/storybook

**Review by GuillaumeNury — APPROVED** [review]

**Pull request merged**